packages/rum-core/src/domain/action/computeFrustration.spec.ts

```diff
@@ -68,6 +68,13 @@ describe('computeFrustration', () => {
       expect(getFrustrations(clicks[1])).toEqual([])
     })
 
+    it('does not add a dead frustration when clicking to scroll', () => {
```

Comment:
**❓ question:** ‏same question: what's the idea behind this?

Reply:
I was trying to cover the case where a sequence of clicks was converted to `rage_click`, and that it should not have `dead_click` if a scroll appeared. But in fact, it can't be converted to a `rage_click` 😅.